docs/ocs-on-a-disconnected-environment.md

```diff
@@ -53,129 +53,150 @@ you should get something similar to this:
+      mediatype: image/svg+xml
+    image: quay.io/noobaa/lib-bucket-catalog@sha256:b9c9431735cf34017b4ecb2b334c3956b2a2322ce31ac88b29b1e4faf6c7fe7d
+    publisher: Red Hat
+    sourceType: grpc  
+  ```
+  
+  create the catalog source:
+  ```
+  oc create -f catalogsource.yaml
+  ```
+
+- Prior to OCS 4.5 for OCP disconnected environments, the lib-bucket-provisioner csv and deployment will need to be edited and the image quay.io/noobaa/pause will need to be replaced with quay.io/noobaa/pause@sha256:b31bfb4d0213f254d361e0079deaaebefa4f82ba7aa76ef82e90b4935ad5b105. 
+After the deployment of OCS, edit the lib-bucket-provisioner csv first with this image@sha. Next, edit the lib-bucket-provisioner deployment and replace quay.io/noobaa/pause with this image@sha if not already correct.
+
+
+## Normal path (after workarounds)
+
+- **After adding the related images (in the `relatedImages workarounds` section), apply the `imageContentSourcePolicy.yaml` file to the cluster**  
+(the file is generated by `oc adm catalog mirror`. the output dir is usually `./[catalog image name]-manifests`)
+  ```
+  oc apply -f ./[output dir]/imageContentSourcePolicy.yaml
+  ```
 
 - **Create CatalogSource for redhat-operators**  
 Create a CatalogSource object that references the catalog image for redhat-operators. Modify the following to your specifications and save it as a catalogsource.yaml file:
-```yaml
-apiVersion: operators.coreos.com/v1alpha1
-kind: CatalogSource
-metadata:
-  name: redhat-operators-catalog
-  namespace: openshift-marketplace
-spec:
-  sourceType: grpc
-  image: yydev2.mirror-registry.qe.gcp.devcluster.openshift.com:5000/test-restricted/redhat-operators:v1 
-  displayName: Redhat Operators Catalog
-  publisher: grpc
-```
-create the catalog source:
-```
-oc create -f catalogsource.yaml
-```
-
-- **repeat for community operators**
-```yaml
-apiVersion: operators.coreos.com/v1alpha1
-kind: CatalogSource
-metadata:
-  name: community-operators-catalog
-  namespace: openshift-marketplace
-spec:
-  sourceType: grpc
-  image: yydev2.mirror-registry.qe.gcp.devcluster.openshift.com:5000/test-restricted/community-operators:v1 
-  displayName: Community Operators Catalog
-  publisher: grpc
-```
+  ```yaml
+  apiVersion: operators.coreos.com/v1alpha1
+  kind: CatalogSource
+  metadata:
+    name: redhat-operators-catalog
```

Comment:
Today I've realized that there were some problem with *ClusterLogging* and it seems to be related to the name of the  *Redhat Operators Catalog*. On normal cluster it is named `redhat-operators` not `redhat-operators-catalog` and in *cluster-logging* subscription I've seen following message:
```
# oc get -n openshift-logging subscriptions.operators.coreos.com -o yaml
...
  status:
  ...
    - lastTransitionTime: "2020-06-09T10:03:34Z"
      message: targeted catalogsource openshift-marketplace/redhat-operators missing
      reason: UnhealthyCatalogSourceFound
      status: "True"
      type: CatalogSourcesUnhealthy
  ...
```
I've deleted the `redhat-operators-catalog` and created it freshly with new name `redhat-operators` and it seems to fix the issue with *ClusterLogging*.
So I think we should name it `redhat-operators`.
```suggestion
    name: redhat-operators
```

Reply:
👍  fixed